pkg/cmd/repo/create/create_test.go

```diff
@@ -197,10 +197,12 @@ func Test_createRun(t *testing.T) {
 		errMsg      string
 	}{
 		{
-			name:       "interactive create from scratch with gitignore and license",
-			opts:       &CreateOptions{Interactive: true},
+			name: "interactive create from scratch with gitignore and license",
```

Comment:
@rajhawaldar : any concerns about extending the tests to include a version of this test that overrides `GH_HOST` like [what is done in other tests](https://github.com/search?q=repo%3Acli%2Fcli+path%3A**%2F*_test.go+%22GH_HOST%22&type=code) to ensure a host other than `github.com` results?